assets/helpers/paymentIntegrations/newPaymentFlow/payPalRecurringCheckout.js

```diff
@@ -0,0 +1,136 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { logException } from 'helpers/logger';
+import { routes } from 'helpers/routes';
+import type { Csrf as CsrfState } from 'helpers/csrf/csrfReducer';
+import type { IsoCurrency } from 'helpers/internationalisation/currency';
+
+// ----- Functions ----- //
+
+function loadPayPalRecurring(): Promise<void> {
+  return new Promise((resolve) => {
+    const script = document.createElement('script');
+    script.onload = resolve;
+    script.src = 'https://www.paypalobjects.com/api/checkout.js';
+    if (document.head) {
+      document.head.appendChild(script);
+    }
+  });
+}
+
+// ----- Auxiliary Functions -----//
```

Comment:
no idea, copied & pasted from some old code! I'll delete. I think it may have meant exported vs non-exported functions but that information is available elsewhere